quickstart/configure-mayastor.md

```diff
@@ -91,24 +91,26 @@ kubectl get dsp -n mayastor
 {% tab title="Example Output" %}
 ```text
 NAME             NODE          STATE    POOL_STATUS   CAPACITY      USED   AVAILABLE
-pool-on-node-1   node-1-14944  Online   Online        10724835328   0      10724835328
-pool-on-node-2   node-2-14944  Online   Online        10724835328   0      10724835328
-pool-on-node-3   node-3-14944  Online   Online        10724835328   0      10724835328
+pool-on-node-1   node-1-14944  Created   Online        10724835328   0      10724835328
+pool-on-node-2   node-2-14944  Created   Online        10724835328   0      10724835328
+pool-on-node-3   node-3-14944  Created   Online        10724835328   0      10724835328
 ```
 {% endtab %}
 {% endtabs %}
 
 {% hint style="info" %}
 
-Mayastor-2.0.1 adds two new fields to the DiskPool operator YAML:
+Mayastor- 2.4 will not consist of the **status.state** field and it adds two new fields to the DiskPool operator YAML:
```

Comment:
We can move this to Post upgrade Info section as its related to this upgrade and might seem irrelevent during next release.

Reply:
This we can remove as discussed.